lib/oli/accounts/schemas/user.ex

```diff
@@ -26,6 +26,7 @@ defmodule Oli.Accounts.User do
     field :phone_number_verified, :boolean
     field :address, :string
     field :research_opt_out, :boolean
+    field :state, :map, default: %{}
```

Comment:
User state will be accessible even across different courses and sections. Is this desired? Or should user state be scoped to a course section view enrollment

Reply:
Oh, I see there is one on enrollment as well to cover this case.